packages/wallets/src/components/WalletGradientBackground/WalletGradientBackground.scss

```diff
@@ -648,96 +648,11 @@
-                }
-                &-dark {
-                    background: radial-gradient(
-                            100% 277.78% at 0% 100%,
-                            rgba(0, 147, 147, 0.24) 0%,
-                            rgba(0, 147, 147, 0.48) 100%
-                        )
-                        #151717;
-                    border-radius: 0.8rem;
-                }
-            }
-
-            &-header {
-                &-light {
-                    background: radial-gradient(
-                            100% 4130.74% at 0% 100%,
-                            rgba(0, 147, 147, 0.4) 0%,
-                            rgba(0, 147, 147, 0.16) 50.52%,
-                            rgba(4, 217, 217, 0.4) 100%
-                        )
-                        #ffffff;
-                }
-                &-dark {
-                    background: radial-gradient(
-                            100% 4130.74% at 0% 100%,
-                            rgba(0, 147, 147, 0.4) 0%,
-                            rgba(0, 147, 147, 0.16) 50.52%,
-                            rgba(4, 217, 217, 0.4) 100%
-                        )
-                        #151717;
-                }
-            }
-        }
-        &-mobile {
-            &-card {
-                &-light {
-                    background: radial-gradient(
-                            160% 190% at 0% 100%,
-                            rgba(0, 147, 147, 0.4) 0%,
-                            rgba(0, 147, 147, 0.5) 50.52%,
-                            rgba(0, 147, 147, 0.48) 100%
-                        )
-                        #ffffff;
-                    border-radius: 0.8rem;
-                }
-                &-dark {
-                    background: radial-gradient(
-                            100% 277.78% at 0% 100%,
-                            rgba(0, 147, 147, 0.24) 0%,
-                            rgba(0, 147, 147, 0.48) 100%
-                        )
-                        #151717;
-                    border-radius: 0.8rem;
-                }
-            }
 
-            &-header {
-                &-light {
-                    background: radial-gradient(
-                            100% 4130.74% at 0% 100%,
-                            rgba(0, 147, 147, 0.4) 0%,
-                            rgba(0, 147, 147, 0.16) 50.52%,
-                            rgba(4, 217, 217, 0.4) 100%
-                        )
-                        #ffffff;
-                }
-                &-dark {
-                    background: radial-gradient(
-                            100% 4130.74% at 0% 100%,
-                            rgba(0, 147, 147, 0.4) 0%,
-                            rgba(0, 147, 147, 0.16) 50.52%,
-                            rgba(4, 217, 217, 0.4) 100%
-                        )
-                        #151717;
-                }
-            }
-        }
-    }
-    &--USDT {
+    &--UST,
+    &--USDT,
+    &--EUSDT,
+    &--TUSDT {
```

Comment:
These all have the same stylings, so I just group them together
<img width="1423" alt="Screenshot 2024-04-17 at 2 58 23 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/72e17727-899d-4ac0-b093-572215dbbda6">